plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -4076,8 +4059,7 @@ private void handleTargets(boolean add, List<Map<String, String>> targets, HostM
         if (targets != null && targets.size() > 0) {
             try {
                 _storageProcessor.handleTargetsForHost(add, targets, host);
-            }
-            catch (Exception ex) {
+            } catch (Exception ex) {
```

Comment:
Can't we use a more specific exception?

Reply:
useful comment, again but not to this PR and this goes for most below as well